apps/hash-graph/lib/graph/src/store/postgres/query/condition.rs

```diff
@@ -27,6 +31,7 @@ pub enum EqualityOperator {
 }
 
 impl Transpile for Condition {
+    #[expect(clippy::too_many_lines)]
```

Comment:
Maybe a sign to refactor? I could imagine a function like `fn transpile_binary_op(lhs: impl Transpile, op: &'static str, rhs: impl Transpile)` would remove quite a lot of repetitive code.

Reply:
I'm about to refactor this function at some point but it's currently pretty straightforward to read.